### ISSUE TITLE
(Docs - Concepts)  Optimizers

### ISSUE THREAD

**Issue by guenthermi** [jina-ai/finetuner#727]
- Reference Optimizers from PyTorch and name the some of the most important once (link to PyTorch documentation for details)
- Explain Optimizer Options
- Explain like layer-wise lr decay
- Explain learning rate schedulers and their options
- (Test if lr schedulers and layer-wise lr decay can be used together and if not mention / explain this)
